modules/social_features/social_post/js/keycode-submit.js

```diff
@@ -24,7 +24,7 @@
           // post forms this has some side effects that are required for a
           // successful submission.
           if ($submit.length) {
-            $submit.click();
+            $submit.mousedown();
```

Comment:
Hi @sjoerdvandervis 
Have you checked if this actually doesn't cause any regression without Ajax comments enabled? 
Click doesn't work probably due to the ajax_comments module stopping propagation, I'm not sure why but it sounds like there was a thought behind it. 

Just curious :)